### DISCUSSION TITLE
Pull back and display response Headers

### DISCUSSION THREAD

**Discussion by ebachman24** [grafana/grafana-infinity-datasource#489] [Ideas]
Hello, i'm wondering if it might be possible to pull back response headers from a given request and display _that_ information in dashboards.  For instance, the Git Lab API returns response headers such as total results or total pages in response headers.  It would be nice to be able to grab that information and display it in a dashboard.  

i.e.
http://gitlab.com/api/v4/groups
_returned headers_: Date, Content-Type, X-Total, X-Total-Pages, etc.  

In this case there is valuable data I would look to display.  For other use cases, people might simply want to display the metadata about the calls they are making (which is contained in the headers).

Perhaps it would be a feature allowing the user to 'select response headers' or 'include response headers' in the returned data.

**Comment by maximemoreillon** [comment]
I would also like to have this feature
